cmd/kubectl-gadget/undeploy.go

```diff
@@ -113,7 +113,9 @@ func runUndeploy(cmd *cobra.Command, args []string) error {
 	// Determine the deployment version to decide on undeploy strategy
 	deployedVersion, err := GetDeployedVersion()
 	if err != nil {
-		return fmt.Errorf("determining deployed version: %w", err)
+		fmt.Printf("Error determining deployed version: %s. Using modern undeploy method...\n", err)
```

Comment:
Should this mention something like "assuming v0.43.0+"? "modern undeploy method" is hard to understand.